### PR TITLE
added aws bootstrap files

### PR DESCRIPTION
**Pull request by andersla**

### CODE REVIEW

**Comment by mcapuccini** [comment]
@andersla could you include documentation for this?

File: terraform.tfvars.aws-template [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# Node configuration
+node_count = "2"
+node_instance_type = "t2.micro"
+
+# Edge configuration
+edge_count = "1"
+edge_instance_type = "t2.micro"
+
+# All node configuration
+disk_size = "10" # Size in GB
```
Comment (mcapuccini) [inline terraform.tfvars.aws-template]: please make this per edge, node, master like I did in GCE

File: terraform.tfvars.aws-template [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+cluster_prefix = "kube-now" # Your cluster prefix
+kubenow_image_id = "kube-now" # Name of the image created with Packer
+kubeadm_token = "your-kubeadm-token" # You can run generate_kubetoken.sh to create a valid token
+
+aws_access_key_id = "your-acces-key-id"
+aws_secret_access_key = "your-access-key-here"
+aws_region = "eu-central-1" # Region for your project
+availability_zone = "eu-central-1a" # Zone where cluster should be created
+
+ssh_user = "ubuntu" # the ssh user that will log on to the nodes
```
Comment (mcapuccini) [inline terraform.tfvars.aws-template]: this should default to ubuntu in main.tf and be removed from here (look in GCE)

**Comment by andersla** [comment]
I forgot to create the pull request,
      now there is one for the doc
      
      
      On 2016-10-27 18:39, Marco Capuccini wrote:
    
    
      @andersla could you include
        documentation for this?
      —
        You are receiving this because you were mentioned.
        Reply to this email directly, view
          it on GitHub, or mute
          the thread.
      
        
          
          
        
        
      
      {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/mcapuccini/KubeNow","title":"mcapuccini/KubeNow","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/mcapuccini/KubeNow"}},"updates":{"snippets":[{"icon":"PERSON","message":"@mcapuccini in #12: @andersla could you include documentation for this?"}],"action":{"name":"View Pull Request","url":"https://github.com/mcapuccini/KubeNow/pull/12#issuecomment-256700069"}}}

**Comment by andersla** [comment]
ok branch updated
      /Anders
      
      On 2016-10-27 18:45, Marco Capuccini wrote:
    
    
      @mcapuccini requested changes on this pull request.
      @andersla only a couple of minor
        things to change
      
      In terraform.tfvars.aws-template:
      > +ssh_user = "ubuntu" # the ssh user that will log on to the nodes
+ssh_keypair_name = "your-ssh-keypair-name" # Name of a keypair to use  (for ssh node access)
+
+# Master configuration
+master_instance_type = "t2.micro" 
+
+# Node configuration
+node_count = "2"
+node_instance_type = "t2.micro"
+
+# Edge configuration
+edge_count = "1"
+edge_instance_type = "t2.micro"
+
+# All node configuration
+disk_size = "10" # Size in GB

      please make this per edge, node, master like I did in GCE
      
      In terraform.tfvars.aws-template:
      > @@ -0,0 +1,26 @@
+# Cluster configuration
+cluster_prefix = "kube-now" # Your cluster prefix
+kubenow_image_id = "kube-now" # Name of the image created with Packer
+kubeadm_token = "your-kubeadm-token" # You can run generate_kubetoken.sh to create a valid token
+
+aws_access_key_id = "your-acces-key-id"
+aws_secret_access_key = "your-access-key-here"
+aws_region = "eu-central-1" # Region for your project
+availability_zone = "eu-central-1a" # Zone where cluster should be created
+
+ssh_user = "ubuntu" # the ssh user that will log on to the nodes

      this should default to ubuntu in main.tf and be removed from
        here (look in GCE)
      —
        You are receiving this because you were mentioned.
        Reply to this email directly, view
          it on GitHub, or mute
          the thread.
      
        
          
          
        
        
      
      {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/mcapuccini/KubeNow","title":"mcapuccini/KubeNow","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/mcapuccini/KubeNow"}},"updates":{"snippets":[{"icon":"PERSON","message":"@mcapuccini requested changes on #12"}],"action":{"name":"View Pull Request","url":"https://github.com/mcapuccini/KubeNow/pull/12#pullrequestreview-6099498"}}}

**Comment by mcapuccini** [comment]
@andersla  the packer build file for aws is missing, I'll test it as soon as you add it

**Comment by andersla** [comment]
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/mcapuccini/KubeNow","title":"mcapuccini/KubeNow","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/mcapuccini/KubeNow"}},"updates":{"snippets":[{"icon":"PERSON","message":"@mcapuccini in #12: @andersla  the packer build file for aws is missing, I'll test it as soon as you add it"}],"action":{"name":"View Pull Request","url":"https://github.com/mcapuccini/KubeNow/pull/12#issuecomment-257163050"}}}
    
    ok is there now

**Comment by andersla** [comment]
Now it is there finally... sometimes I forget the "commit" after the "add" of a file (in my brain a file should be in the branch when I add it:)

**Pull request merged**